restonomer-spark-utils/src/main/scala/com/clairvoyant/restonomer/spark/utils/transformer/DataFrameTransformerImplicits.scala

```diff
@@ -63,6 +63,25 @@ object DataFrameTransformerImplicits {
     def replaceStringInColumnValue(columnName: String, pattern: String, replacement: String): DataFrame =
       df.withColumn(columnName, regexp_replace(col(columnName), pattern, replacement))
 
+    def addPrefixToColNames(prefix: String, columnNames: List[String]): DataFrame = {
+
+      if (columnNames == null || columnNames.isEmpty)
+        df.select(
+          df.columns.map(columnName => df(columnName).alias(prefix + columnName)): _*
```

Comment:
Add underscore after prefix

Reply:
Underscore will now be getting added automatically, separating prefix and actual column name. User don't need to provide it.